lib/mastermind/game.rb

```diff
@@ -23,11 +23,13 @@ def over?
     end
 
     def get_exact_matches
-      @exact_matches = @input.get_user_input
+      @exact_matches = @display.get_exact_matches
```

Comment:
Maintaining state here probably just for testability. Should these methods be exposed on the public API of the Game class? I'd avoid this type of mutable state as much as possible as you'll get temporal couplings (i.e., must call get_exact_matches before calling do_something_that_depends_on_exact_matches.